sites/public/src/components/listings/search/ListingsSearchCombined.tsx

```diff
@@ -28,21 +33,29 @@ function ListingsSearchCombined(props: ListingsSearchCombinedProps) {
   const { profile, listingsService } = useContext(AuthContext)
   const [modalOpen, setModalOpen] = useState(false)
   const [filterCount, setFilterCount] = useState(0)
-
-  // Store the current search params for pagination
-  const searchParams = useRef({
-    bedrooms: null,
-    bathrooms: null,
-    monthlyRent: null,
-    counties: [],
-  } as ListingSearchParams)
+  const [listView, setListView] = useState<boolean>(true)
+  const [visibleMarkers, setVisibleMarkers] = useState<MapMarkerData[]>(null)
+  const [currentMarkers, setCurrentMarkers] = useState<MapMarkerData[]>(null)
+  const [isDesktop, setIsDesktop] = useState(true)
+  const [isLoading, setIsLoading] = useState(true)
+  const [searchFilter, setSearchFilter] = useState(
+    parseSearchString(props.searchString, {
```

Comment:
Set the initial search as the search from the query params (as if you were coming from the homepage filter modal)